packages/website/ts/components/fill_order.tsx

```diff
@@ -525,20 +518,21 @@ export class FillOrder extends React.Component<FillOrderProps, FillOrderState> {
             globalErrMsg = 'You must specify a fill amount';
         }
 
-        const signedOrder = this.props.blockchain.portalOrderToSignedOrder(
-            parsedOrder.signedOrder.maker,
-            parsedOrder.signedOrder.taker,
-            parsedOrder.signedOrder.makerTokenAddress,
-            parsedOrder.signedOrder.takerTokenAddress,
-            new BigNumber(parsedOrder.signedOrder.makerTokenAmount),
-            new BigNumber(parsedOrder.signedOrder.takerTokenAmount),
-            new BigNumber(parsedOrder.signedOrder.makerFee),
-            new BigNumber(parsedOrder.signedOrder.takerFee),
-            new BigNumber(this.state.parsedOrder.signedOrder.expirationUnixTimestampSec),
-            parsedOrder.signedOrder.feeRecipient,
-            parsedOrder.signedOrder.ecSignature,
-            new BigNumber(parsedOrder.signedOrder.salt),
-        );
+        const signedOrder = {
+            exchangeContractAddress: this.props.blockchain.getExchangeContractAddressIfExists(),
```

Comment:
`getExchangeContractAddressIfExists` could return undefined. Handle this case correctly.

Reply:
It will return undefined only on unsupported network. As far as I remember portal shows an error message in that case so it's handled already.